doc/source/whatsnew/v0.19.0.txt

```diff
@@ -673,6 +673,7 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Bug in ``pandas.parser.parser_trim_buffers()``, which did not update the word vectors (``parser->words``) when stream buffer was shrunk (:issue:`13788`, :issue:`13703`)
```

Comment:
bug in ``pd.read_csv`` causing a segfault, when ........

 This is read by users.